site_root/input/ImplementationGuide-NCPI-Project-Forge.json

```diff
@@ -55,91 +55,123 @@
           "reference": "StructureDefinition/us-core-ethnicity"
         }
       },
       {
-        "description": "NCPI SearchParameter us-core-ethnicity",
+        "description": "NCPI Project Forge SearchParameter us-core-ethnicity",
         "exampleBoolean": false,
-        "name": "NCPI SearchParameter/us-core-ethnicity",
+        "name": "NCPI Project Forge SearchParameter/us-core-ethnicity",
         "reference": {
           "reference": "SearchParameter/us-core-ethnicity"
         }
       },
       {
-        "description": "NCPI SearchParameter us-core-race",
+        "description": "NCPI Project Forge SearchParameter us-core-race",
         "exampleBoolean": false,
-        "name": "NCPI SearchParameter/us-core-race",
+        "name": "NCPI Project Forge SearchParameter/us-core-race",
         "reference": {
           "reference": "SearchParameter/us-core-race"
         }
       },
       {
-        "description": "NCPI CodeSystem cdcrec",
+        "description": "NCPI Project Forge CodeSystem cdcrec",
         "exampleBoolean": false,
-        "name": "NCPI CodeSystem/cdcrec",
+        "name": "NCPI Project Forge CodeSystem/cdcrec",
         "reference": {
           "reference": "CodeSystem/cdcrec"
         }
       },
       {
-        "description": "NCPI ValueSet detailed-ethnicity",
+        "description": "NCPI Project Forge ValueSet detailed-ethnicity",
         "exampleBoolean": false,
-        "name": "NCPI ValueSet/detailed-ethnicity",
+        "name": "NCPI Project Forge ValueSet/detailed-ethnicity",
         "reference": {
           "reference": "ValueSet/detailed-ethnicity"
         }
       },
       {
-        "description": "NCPI ValueSet detailed-race",
+        "description": "NCPI Project Forge ValueSet detailed-race",
         "exampleBoolean": false,
-        "name": "NCPI ValueSet/detailed-race",
+        "name": "NCPI Project Forge ValueSet/detailed-race",
         "reference": {
           "reference": "ValueSet/detailed-race"
         }
       },
       {
-        "description": "NCPI ValueSet omb-ethnicity-category",
+        "description": "NCPI Project Forge ValueSet omb-ethnicity-category",
         "exampleBoolean": false,
-        "name": "NCPI ValueSet/omb-ethnicity-category",
+        "name": "NCPI Project Forge ValueSet/omb-ethnicity-category",
         "reference": {
           "reference": "ValueSet/omb-ethnicity-category"
         }
       },
       {
-        "description": "NCPI ValueSet omb-race-category",
+        "description": "NCPI Project Forge ValueSet omb-race-category",
         "exampleBoolean": false,
-        "name": "NCPI ValueSet/omb-race-category",
+        "name": "NCPI Project Forge ValueSet/omb-race-category",
         "reference": {
           "reference": "ValueSet/omb-race-category"
         }
       },
       {
-        "description": "NCPI Patient pt-001, Profiles: ncpi-patient",
+        "description": "NCPI Project Forge Patient pt-001, Profiles: ncpi-patient",
```

Comment:
It seems `Profiles: ncpi-patient` should be `Profiles: Patient` because we remove the `ncpi-patient` profile in this PR. Accordingly, in [the example resource](https://github.com/ncpi-fhir/ncpi-model-forge/blob/simplified-model/site_root/input/resources/examples/Patient-pt-001.json), we need to change this line (https://github.com/ncpi-fhir/ncpi-model-forge/blob/simplified-model/site_root/input/resources/examples/Patient-pt-001.json#L6) to `"http://hl7.org/fhir/StructureDefinition/Patient"`

Reply:
Completely overlooked the example files. Updated the branch with the requested change.